### PR TITLE
[Tests-Only] Add acceptance test case for uploading files with maximum file name length possible

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This PR adds acceptance tests for syncing files with UTF-8 names, where files with maximum possible filename length are uploaded using chunking.

## Related Issue
- https://github.com/owncloud/core/issues/39095

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiWebdavUpload2/uploadFileUsingNewChunking.feature [hunk 1]
```diff
@@ -168,3 +168,21 @@ Feature: upload file using new chunking
...
+
+
+  Scenario: Upload chunked file with new chunking with lengthy filenames
+    Given the owncloud log level has been set to debug
+    And the owncloud log has been cleared
+    When user "Alice" uploads the following chunks to "हजार नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-012345.txt" with new chunking and using the WebDAV API
+      | number | content |
+      | 1      | AAAAAAAAAAAAAAAAAAAAAAAAA |
+      | 2      | BBBBBBBBBBBBBBBBBBBBBBBBB  |
+      | 3      | CCCCCCCCCCCCCCCCCCCCCCCCC   |
```
Comment (Talank) [inline tests/acceptance/features/apiWebdavUpload2/uploadFileUsingNewChunking.feature:180]: indentation does not seem correct 🤔

File: tests/acceptance/features/apiWebdavUpload2/uploadFileUsingOldChunking.feature [hunk 2]
```diff
@@ -99,3 +99,21 @@ Feature: upload file using old chunking
...
       | ?abc=oc #   |
+
+  Scenario: Upload chunked file with old chunking with lengthy filenames
+    Given the owncloud log level has been set to debug
+    And the owncloud log has been cleared
+    When user "Alice" uploads the following chunks to "नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-नेपालि-file-नाम-12345678910.txt" with old chunking and using the WebDAV API
+      | number | content |
+      | 1      | AAAAAAAAAAAAAAAAAAAAAAAAA |
+      | 2      | BBBBBBBBBBBBBBBBBBBBBBBBB  |
+      | 3      | CCCCCCCCCCCCCCCCCCCCCCCCC   |
```
Comment (Talank) [inline tests/acceptance/features/apiWebdavUpload2/uploadFileUsingOldChunking.feature:110]: same here

**Comment by individual-it** [comment]
> This PR adds acceptance tests for syncing files with UTF-8 names, where files with maximum possible filenames are uploaded using chunking.

you mean maximum possible filename length?

File: tests/acceptance/features/apiWebdavUpload2/uploadFileUsingOldChunking.feature (hunk 2 above)
Comment (jasson99) [reply tests/acceptance/features/apiWebdavUpload2/uploadFileUsingOldChunking.feature]: fixed

File: tests/acceptance/features/apiWebdavUpload2/uploadFileUsingNewChunking.feature (hunk 1 above)
Comment (jasson99) [reply tests/acceptance/features/apiWebdavUpload2/uploadFileUsingNewChunking.feature]: fixed

**Review by individual-it — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]
